devenv/src/main.rs

```diff
@@ -268,6 +271,7 @@ struct App {
     has_processes: Option<bool>,
     container_name: Option<String>,
     assembled: bool,
+    dirs_created: Option<bool>,
```

Comment:
A `bool` would do the job here. Is there a reason to to wrap it in an `Option`?
```suggestion
    dirs_created: bool,
```

Reply:
Yeah, you are right. I was just following similar patterns as `assemble`, fixed everything in the latest commit.